lms/djangoapps/grades/models.py

```diff
@@ -257,8 +257,14 @@ def save_grade(cls, **kwargs):
                     usage_key=usage_key,
                     defaults=kwargs,
                 )
+            log.info("persistent subsection grade successfully saved: {0}".format(unicode(grade)))
```

Comment:
Can we instead update the `unicode` method to serialize `self.visible_blocks_id`?  That shouldn't require an extra sql query.